.github/workflows/ci.yml

```diff
@@ -3,22 +3,16 @@
 
 name: Build and test CI
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push]
```

Comment:
Would be nice to also scheduled this job every day at 0:00 (for instance) to be sure any new livestorm release don’t break the SDK.